whitepapers/business-types.md

```diff
@@ -1,12 +1,12 @@
 ---
 published: true
-permalink: /whitepapers/business-type/
+permalink: /whitepapers/business-types/
```

Comment:
The links on the data elements are updated but I will update the GitHub issue when it goes live on Monday.